src/OIDCCryptUtils.purs

```diff
@@ -0,0 +1,48 @@
+module OIDCCryptUtils (RSASIGNTIME(), hashNonce, bindState, unbindState, verifyIdToken) where
```

Comment:
`newtype`s are generally better anyway though, even with a record representation, as they keep strings separated by their semantic category, so you can't ever accidentally provide a nonce where it was supposed to be an idtoken, for example.